### ISSUE TITLE
Sentry suddenly losing environment

### ISSUE THREAD

**Issue by Jamesking56** [getsentry/sentry-php#1231]
### Environment

How do you use Sentry?
Sentry SaaS (sentry.io)

Which SDK and version?
`sentry/sentry` `1.11.0`

### Steps to Reproduce

1. Create a new `\Raven_Client()`
2. Set the environment with `->setEnvironment()`
3. Send an exception

### Expected Result

Environment comes through to the new logged Sentry error

### Actual Result

The environment is missing. We haven't changed our Sentry implementation and suddenly the environment is no longer being set (causing us not to receive alerts either)

**Comment by stayallive** [comment]
Hi @Jamesking56, I've did the following to test this out:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

$sentry = new Raven_Client('https://xxx@o123456.ingest.sentry.io/123456');

$sentry->setEnvironment('prod');

$sentry->captureException(
    new Exception('This is a test on the prod environment.')
);
```

And the event comes in with the environment correctly set to `prod`.

Are the problems being captured maybe occurring before the `setEnvironment` call or are you using a service container that creates multiple instances of the `Raven_Client`?

I must also note that `1.x` is a very old SDK and not really supported, my guess is that you have a good reason for using it but if at all possible it might be a good time to upgrade even though from my testing nothing looks to be broken in `1.x`.

**Comment by Jamesking56** [comment]
We are using Symfony 4, so it could be something to do with the service container there. We are planning to upgrade to `3.x` eventually, we have a lot of apps still using `1.x` though.

Still investigating it at the moment, we have other apps on `1.x` which are still having the environment come through fine, so trying to work out why this app isn't

**Comment by stayallive** [comment]
Interesting, you might want to consider giving the Symfony bundle a try when upgrading anyway. 

If you figure it out let us know but I'm leaning towards some container issues possibly especially since you are also confirming other apps have no issues and I'm unable to verify the issue too as mentioned previously. 

If I can help with anything please let me know 👍

**Comment by Jamesking56** [comment]
@stayallive Managed to fix this, we changed the name of some items in our Symfony container configuration - including Sentry. By mistake, we didn't change some references which used the old name.

So yes, some places in the app were getting an unconfigured `Raven_Client` instead of a configured one.

I've got a working fix for our app tested with Sentry. Can confirm no bug in the `1.x` library.

Thanks for investigating your side for me :+1:
